### DISCUSSION TITLE
Any plan to add built-in image component?

### DISCUSSION THREAD

**Discussion by kawpii** [solidjs/solid#841] [Q&A] [answered]
Like in [Gatsby Image](https://www.gatsbyjs.com/docs/how-to/images-and-media/using-gatsby-plugin-image/), [Next Image](https://nextjs.org/docs/api-reference/next/image)

**Comment by ryansolid** [comment, marked as answer]
Possibly but I don't think it would be core. That stuff depends on deployment platform to my knowledge. I have experience writing these sort of components having worked in image sharing/social media for the better part of a decade but it does seem like a tertiary things. Something that SolidStart or maybe even Solid Primitives might be interested in.
